.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
     "cmake.configureOnOpen": true,
     "cmake.configureSettings": {
-        "CMAKE_TOOLCHAIN_FILE": "${workspaceFolder}/3rdparty/vcpkg/scripts/buildsystems/vcpkg.cmake"
+        //"CMAKE_TOOLCHAIN_FILE": "${workspaceFolder}/3rdparty/vcpkg/scripts/buildsystems/vcpkg.cmake"
```

Comment:
Um, I was under the impression that comments are not permitted in JSON (they once were as I understand it) e.g. see https://www.freecodecamp.org/news/json-comment-example-how-to-comment-in-json-files/ ...

Reply:
Some parsers still accept them